src/tactic/interactive.lean

```diff
@@ -613,7 +621,11 @@ example : foo :=
 by use [100, tt, 4, 3]
 -/
 meta def use (l : parse pexpr_list_or_texpr) : tactic unit :=
-tactic.use l >> try triv
+focus1 $ do tactic.use l,
```

Comment:
This is not correct. `tactic.use` can create multiple subgoals, so the `focus1` isn't doing what you expect.

Reply:
Hmm, maybe it doesn't matter if both of the `try`s succeed on goals generated by `use`. I need to run and don't have time to confirm this right this sec -- if you're sure that it's safe, you can add the ready to merge flag back on.